application/config.go

```diff
@@ -0,0 +1,61 @@
+package application
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/BurntSushi/toml"
+	"github.com/coniks-sys/coniks-go/crypto/sign"
+	"github.com/coniks-sys/coniks-go/utils"
+)
+
+// AppConfig is the generic type used to specify the configuration of
+// any kind of CONIKS application-level executable (e.g. key server,
+// client etc.).
+type AppConfig interface {
+	InitConfig(file string) error
```

Comment:
Could be just `Init`, or `Load`. I think the latter is more precise.

Reply:
Hmm, I thought of this. There already is a `Load` function in `application/config`, so I didn't want to have double-naming to reduce confusion. I also decided to include the `Config` for readability of the code: `server.LoadConfig()` vs `server.Load()`; I find that the former is a bit more specific. What do you think?